client/src/components/CommentComponent/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Comment, Form } from 'semantic-ui-react';
+import CommentReply from '../CommentReply/index';
+
+const CommentComponent = ({
+  image, author, postedOn, content,
+}) => {
+  const [showReplyBox, setReplyBoxState] = useState(false);
+  const [singleReply, setSingleReply] = useState(null);
+  const [replies, setReplies] = useState([]);
+
+  const createReplyComment = (event, replyContent) => {
+    event.preventDefault();
+    setReplyBoxState(false);
+    setSingleReply('');
+    if (!replyContent || replyContent.trim() === '') {
+      return;
+    }
+    if (replies) {
+      setReplies(replies.concat(
+        <CommentReply
+          image={image}
+          author={author}
+          postedOn={new Date().toString()}
```

Comment:
write a function which converts date into relative time like few seconds earlier, few minutes ago etc

check this https://stackoverflow.com/a/6109105